scipy/interpolate/tests/test_interpolate.py

```diff
@@ -322,11 +324,32 @@ def test_nearest(self):
                     bounds_error=True)
         assert_raises(ValueError, interp1d, self.x10, self.y10, **opts)
 
+    def test_nearest_up(self):
+        # Check the actual implementation of nearest-neighbour interpolation.
```

Comment:
Can/should expand this comment, and again stress the difference between nearest and nearest-up

Reply:
Fixed.